test/data/conformance/australian-vet.xml

```diff
@@ -28,6 +28,8 @@
             <eeSummary:city>Sydney</eeSummary:city>
             <eeSummary:country>AUS</eeSummary:country>
             <eeSummary:line1>123 NW 8th St</eeSummary:line1>
+            <eeSummary:line2></eeSummary:line2>
```

Comment:
Based on how we treat null/non-existant elements elsewhere, I'm thinking that these shouldn't be in the generated XML if they're blank